services/121-service/src/main.ts

```diff
@@ -71,6 +73,39 @@ function generateModuleDependencyGraph(app: INestApplication): void {
   });
 }
 
+function generateSwaggerSummaryJson(app: INestApplication<any>): void {
+  const options = new DocumentBuilder()
+    .setTitle(APP_TITLE)
+    .setVersion(APP_VERSION)
+    .build();
+  const openApiDocument = SwaggerModule.createDocument(app, options);
+
+  const minimalDocument: {
+    method: string;
+    path: string;
+    returnType: string;
+  }[] = [];
+
+  for (const path in openApiDocument.paths) {
+    for (const method in openApiDocument.paths[path]) {
+      const methodInfo = openApiDocument.paths[path][method];
+      const returnType =
+        methodInfo.responses['200']?.content?.['application/json']?.schema?.$ref
+          ?.split('/')
+          .pop() || 'Not set/ undefined';
```

Comment:
```suggestion
          .pop();
```
related to my other comment above.